src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravVarselService.kt

```diff
@@ -18,12 +17,10 @@ import java.util.*
 class AktivitetskravVarselService(
     private val aktivitetskravVarselRepository: AktivitetskravVarselRepository,
     private val aktivitetskravVurderingProducer: AktivitetskravVurderingProducer,
-    private val arbeidstakervarselProducer: ArbeidstakervarselProducer,
     private val aktivitetskravVarselProducer: AktivitetskravVarselProducer,
     private val expiredVarselProducer: ExpiredVarselProducer,
     private val pdfGenClient: PdfGenClient,
     private val pdlClient: PdlClient,
-    private val krrClient: KRRClient,
```

Comment:
Jeg tror denne egentlig aldri var i bruk 🤔 Vi hadde en PR som satt opp KRRClient, men så fant vi ut at vi egentlig ikke trengte å sjekke om bruker var `reservert` likevel, hvertfall ikke per nå. Så det er kanskje greit at vi bare fjerner den igjen, @navikt/teamsykefravr?

Reply:
Jeg er fortsatt litt usikker på om det er noe vi trenger å sjekke for å sette riktig kanal på journalposten når vi journalfører i `JournalforAktivitetskravVarselCronjob` :thinking: 
Har eSyfo noe logikk på sin side som sjekker `reservert` nå eller bruker dere bare `distribuerJournalpost`-tjenesten?

Reply:
Det digitale varselet sendes kun til **ikke** reserverte brukere. Bruker-status sjekkes før utsending og om bruker **er** reservert, sendes det kun fysisk brev (dvs `distribuerJournalpost`).

Reply:
Ok. Sånn jeg har skjønt det så vil `distribuerJournalpost` sette riktig kanal på journalposten ut i fra distribusjonsmåten den velger. Men burde vi da sette `DITT_NAV` eller noe sånt for de som er `ikke reserverte`? Vi gjør det for dialogmøtevarsler, men vet ikke om det har noe å si :man_shrugging:

Reply:
Det er jeg usikker på. Bare om å presisere esifovarse distribuerer ikke noe for ikke reserverte

Reply:
Virker kanskje best om vi ikke setter noen kanal i journalføringen her, så kan dokarkiv ta seg av det. Uansett greit å slette KRRClient her da :smiley: